CONTRIBUTING.md

```diff
@@ -317,8 +317,8 @@ The Runtimes for all supported cloud functions (multicore builds) can be found a
 
 For generating a new console SDK follow the next steps:
 
-1. Update the console spec file located at `app/config/specs/swagger2-0.12.x.console.json` from the dynamic version located at `https://localhost/specs/swagger2?platform=console`
-2. Generate a new SDK using the command `php app/cli.php sdks`
+1. Update the console spec file located at `app/config/specs/swagger2-<version-number>.console.json` using the Appwrite CLI tools. Run the `php app/cli.php specs <version-number> normal` command in a running `appwrite/appwrite` container..
```

Comment:
> Appwrite CLI

This might be confusing considering we have [Appwrite CLI](https://github.com/appwrite/sdk-for-cli).

```suggestion
1. Update the console spec file located at `app/config/specs/swagger2-<version-number>.console.json` using the `specs` task. Run the `php app/cli.php specs <version-number> normal` command in a running `appwrite/appwrite` container.
```

Reply:
Agree with Steven. Internally we usually refer them as Appwrite Tasks.